src/attribution/onCLS.ts

```diff
@@ -81,7 +81,7 @@ export const onCLS = (
   opts?: ReportOpts
 ) => {
   unattributedOnCLS(
-    ((metric: CLSMetric) => {
+    ((metric: CLSMetricWithAttribution) => {
```

Comment:
This looks like it was missed previously. All the other metrics apart from CLS have this set correctly.

Reply:
> This looks like it was missed previously. All the other metrics apart from CLS have this set correctly.